### ISSUE TITLE
"New props" arrow label is misleading

### ISSUE THREAD

**Issue by stephan-noel** [wojtekmaj/react-lifecycle-methods-diagram#39]
### Motivation
I realize that the phrase "New props" is in the original tweet of Dan Abramov, but "new props" is vague and misleading.  Further, it tends to reinforce [the misconception mentioned in Brian Vaughn's blog post](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#anti-pattern-unconditionally-copying-props-to-state) that the "Updating" swimlane is triggered **only** when the props "change" or are "different" ( "new" props suggests "different" from "old" props).  As the blog post says "_these lifecycles are called any time a parent component rerenders, regardless of whether the props are 'different' from before_".

### Proposal
Change the wording to say "parent rerenders" or something similar.  

### Aside
I'm not trying to be picky.  Dan's diagram and your project have been infinitely helpful in aiding my understanding, yet I've continuously seen beginner, intermediate, and even some advanced React developers tripped up by this confusion.  It's a small change, but if you agree, I can help out and do the PR as well.

**Comment by csr632** [comment]
Agreed. "New props" is confusing and doesn't have any (correct) meaning. "Parent rerenders" give infomation about "why" and "when", just like the "set­State()" and "force­Update()" label.

**Comment by stephan-noel** [comment]
Thanks for your input @csr632 !  In the meantime, I've had a colleague suggest "nextProps".  Also, maybe "parent render()" would be preferable to "parent rerenders" to clarify the possible ambiguity between the React lifecycle render() method vs. the browser rendering.

**Comment by ReactAcademyCa** [comment]
I feel that "parent render()" is perfect.  

When I give a course, I use this wording:
**Render:**  the method to generate the virtual DOM. (generates a tree structure of React Elements, which are simple js objects).
**Reconcilation:** the React process to update the DOM. (React does this by comparing the current vDOM with the previous one.)
**Browser paint** or **Browser refresh**: the browser process to show the DOM on the screen.
